src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-
     <TargetFrameworkNetDesktop>net462</TargetFrameworkNetDesktop>
     <TargetFrameworkNetCore>netcoreapp3.1</TargetFrameworkNetCore>
+    <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
 
     <PlatformTarget>AnyCPU</PlatformTarget>
-    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">$(TargetFrameworkNetDesktop);$(TargetFrameworkNetCore)</TargetFrameworks>
-    <TargetFramework Condition="$([MSBuild]::IsOsPlatform('OSX'))">$(TargetFrameworkNetCore)</TargetFramework>
-    <TargetFramework Condition="$([MSBuild]::IsOsPlatform('Linux')) ">$(TargetFrameworkNetCore)</TargetFramework>
+    <TargetFramework Condition="$([MSBuild]::IsOsPlatform('Windows'))">$(TargetFrameworkNetStandard)</TargetFramework>
```

Comment:
nit: There's no difference between lines 8, 9 and 10. You can simply have ` <TargetFramework>$(TargetFrameworkNetStandard)</TargetFramework>`